lib/components/live_helpers.ex

```diff
@@ -16,9 +16,12 @@ defmodule CrunchBerry.Components.LiveHelpers do
   ## Options
 
   - `id` - required.  The modal is a `Phoenix.LiveComponent`, and needs a specified `id`.
-  - `return_to` - required.  This is the route that will be pushed to when the modal is closed,
-     either by the "x" or clicking the background.
+  - `return_to` - optional.  This is the route that will be pushed to when the modal is closed,
+     either by the "x" or clicking the background. Either return_to or phx_target must be specified.
```

Comment:
If neither `return_to` or `phx_target` is specified, the events will just go to the parent live view, which I think is OK.

Reply:
updated